server/deploy/helm/charts/lomas_server/templates/server/deployment.yaml

```diff
@@ -30,7 +30,7 @@ spec:
         imagePullPolicy: {{ .Values.server.image.pullPolicy }}
         ports:
           - name: http
-            containerPort: {{ .Values.server.service.port }}
+            containerPort: {{ .Values.server.runtime_args.settings.server.host_port }}
```

Comment:
I am trying to wrap my head around this one. Did it work because service.port was set to 80 and host port was http in the templates so since everything was 80 it so happened that it was ok?

Reply:
Yes, i think it was very lucky because everything was 80